InterfaceStubGenerator.Core/InterfaceStubGenerator.cs

```diff
@@ -448,6 +447,27 @@ public class MethodTemplateInfo
         public string InterfaceName { get; set; }
         public List<string> TypeParametersInfo { get; set; }
         public string TypeParameters => TypeParametersInfo != null ? string.Join(", ", TypeParametersInfo) : null;
+
+        public string Id
+        {
+            get
+            {
+                HashCode hashCode = default;
+
+                hashCode.Add(IsRefitMethod);
+                hashCode.Add(Name);
+                hashCode.Add(ReturnType);
+                hashCode.Add(ArgumentListWithTypes);
+                hashCode.Add(MethodTypeParameters);
+                hashCode.Add(TypeParameters);
+                hashCode.Add(InterfaceName);
+
+                var key = hashCode.ToHashCode();
+
+                // Cast to uint as we can't have an id starting with the '-' character
+                return ((uint)key).ToString();
```

Comment:
I'd suggest changing this to `ToString(CultureInfo.InvariantCulture)`, otherwise you might get different outputs based on the culture setting of the user's computer.

Reply:
Wait, can that happen even for integer values? Shouldn't there be no punctuation or culture-specific stuff at all in this case anyway? 🤔

Reply:
The integer could come out with thousand separators, for example: `1,234,567`

It might also come out in a non-Latin character set (Arabic, Hebrew, Japanese).

Reply:
Ah, well, TIL, thanks! 😄

Done in https://github.com/reactiveui/refit/pull/897/commits/f80fac1496c1765415b073005e58fed74ab718b6.